service/ui.go

```diff
@@ -109,6 +111,17 @@ func NewUIServer(env evergreen.Environment, queue amboy.Queue, home string, fo T
 		hostCache: make(map[string]hostCacheItem),
 	}
 
+	if settings.AuthConfig.Kanopy != nil {
+		uis.umconf.OIDC = &gimlet.OIDCConfig{
```

Comment:
I'm a little confused that I can't find this struct in the gimlet codebase; could you link to this anywhere? Could you explain what these fields are needed for? Should Validate be tweaked to verify these values, if they're included?

Reply:
It's being added in https://github.com/evergreen-ci/gimlet/pull/172.
[This commit](https://github.com/evergreen-ci/gimlet/commit/fba0d893938dda17b34c676c1636f70f548b534c#diff-38132d4f7f8639c7821ad118b974833bf5a02b222d40a00584df8f5474198bfbR30-R42) adds it.

Reply:
Tweaked validate to include the values

Reply:
Added explanation for each of the fields.